### DISCUSSION TITLE
ESP32-H2 RCP OTBR

### DISCUSSION THREAD

**Discussion by aliakseika** [openthread/openthread#9544] [Q&A] [answered]
Hello. Is it possible to connect RCP ESP-32-H2 to ot-otbr-posix via USB? And not use otbr on ESP32-S3? I ask because the default RCP firmware does not work on ot-otbr-posix. 

PS: managed to launch. It was necessary to disable the logs and explicitly specify the port speed spinel+hdlc+uart:///dev/ttyUSB0?uart-baudrate=460800

**Comment by chshu** [comment, marked as answer]
Yes, the default baudrate in [ot_rcp](https://github.com/espressif/esp-idf/tree/master/examples/openthread/ot_rcp) is 460800. Good to know it works now.
